package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ml-levenberg-marquardt",
-  "version": "1.0.3",
+  "version": "1.1.0",
```

Comment:
I think it's customary to separate release bump commits from other work (some package maintainers get really peeved about this). @targos Are there strong opinions about this for this project?

Reply:
Yes, we use a tool that bumps the version number  automatically (based on the commit messages) when we publish the release